_blog/kubernetes-on-solar-plants/index.md

```diff
@@ -17,93 +17,83 @@ js:
+Demand for renewable energy has been growing steadily in the last decade, with solar power outstripping the growth in all other renewable forms of power generation.
 
 Solar panels are getting cheaper, and are becoming an economically viable source of renewable energy in many parts of the world.
 
-**And not only businesses and governments, but also individuals find it attractive to build their solar plants.**
+**In addition to businesses and governments, individuals are increasingly finding that installing solar power arrays is viable at a lower scale.**
 
-Capturing the optimal amount of energy from a solar panel is a tricky business, however.
+Capturing the optimal amount of energy from a solar panel is, however, a tricky business.
 
-If you want your solar panels to operate efficiently, you have to keep them clean, point them at the sun and make sure they don't get too hot.
+In order for solar panels to operate efficiently, they need to be kept clean and pointed at an optimal angle to the sun that balances power generation and prevents overheating.
 
-_So how do you keep everything under control and be notified when something isn't right, and it requires your attention?_
+_So how do you keep everything under control and get notified when something isn't right and requires your attention?_
 
 ## Software is eating the world
 
-Solar panels can feature a small embedded device capable of
+One solution is to capitalise on small embedded devices that
 
-- measuring the performance and efficiency of the cells
-- monitoring the environment
-- driving the actuators to track the sun during the day
+- measure the performance and efficiency of the cells
+- monitor the environment
+- drive the actuators to track the sun during the day
 
-The embedded computer collects and sends the data to a central location where is then aggregated, processed and stored.
+The embedded computer collects data and sends them to a central location where is then aggregated, processed and stored.
 
-If the solar panel is dropping in efficiency, an operator can quickly be alerted and take action.
+If the solar panel is dropping in efficiency, an operator can be alerted to take action.
 
-In larger plants, all the data is passed through a wired network, but it's not uncommon to see those embedded computers connecting over the air.
+In larger plants, the solar arrays' data are passed through a wired network, but it's not uncommon to see those embedded computers connecting wirelessly.
 
-**Imagine controlling your solar panels over 5G:** the beauty of not having to deal with extra cables and the horror of long responses, dropped connections and timeouts.
+**Imagine controlling your solar panels over 5G:** the beauty of not having to deal with extra cables and the horror of long responses <!-- Unsure why long responses would occur -->, dropped connections and timeouts.
```

Comment:
TCP packets could be delayed. As a result, response times could go from milliseconds to seconds.